tests/js/web/TableDetailComponentTest.js

```diff
@@ -115,4 +115,24 @@ describe('TableDetailComponent', () => {
 			expect($body.find('[data-table-detail-panel]').hasClass('table-detail--open')).to.be.false;
 		});
 	})
+
+	describe('When the backdrop overlay is clicked', () => {
+		it('should remove the "in" class from the backdrop to hide it', () => {
+			tableDetailComponent.init($body);
+
+			$body.find('[data-table-detail-view-detail]').trigger(clickEvent);
+			$body.find('.table-detail-backdrop').trigger(clickEvent);
+
+			expect($body.find('.table-detail-backdrop').hasClass('in')).to.be.false;
+		});
+
+		it('should remove the "table-detail--open" class from the panel to close it', () => {
+			tableDetailComponent.init($body);
+
+			$body.find('[data-table-detail-view-detail]').trigger(clickEvent);
+			$body.find('.table-detail-backdrop').trigger(clickEvent);
+
+			expect($body.find('[data-table-detail-panel]').hasClass('table-detail--open')).to.be.false;
```

Comment:
Same as above. We could be confident that both parts of the functionality are working together by asserting that the `table-detail--open` is added when the first click is triggered.